src/pages/partners/affiliate-ib/_faq-data.tsx

```diff
@@ -5,17 +5,31 @@ import { localize, Localize, LocalizedLink } from 'components/localization'
 import { Header, Text, LinkText } from 'components/elements'
 import { affiliate_signup_url } from 'common/constants'
 import { isBrowser } from 'common/utility'
+import device from 'themes/device'
 
 const ItemContainer = styled(SectionContainer)`
     padding: 0;
 `
 
 const HeaderPrimary = styled(Header)`
-    margin-top: 3.2rem;
+    margin-top: 16px;
+    font-size: 16px;
+
+    @media ${device.tabletL} {
+        font-size: 14px;
+        line-height: 20px;
+    }
```

Comment:
We should be able to replace this with `<Header as="p" type="paragraph-1" mt="16px">`.

Reply:
mt property doesn't work. May be because file is .txs?